src/ioctls/vm.rs

```diff
@@ -1942,6 +1943,7 @@ pub(crate) fn request_gic_init(vgic: &DeviceFd) {
         addr: 0,
         flags: 0,
     };
+    assert!(vgic.has_device_attr(&vgic_attr).is_ok());
```

Comment:
Kinda unrelated to your PR, but could we do
```suggestion
    vgic.has_device_attr(&vgic_attr).unwrap();
```
instead? `assert((...).is_ok())` gives very unhelpful error messages, because it doesn't print out the `Err` variant (should an error happen).

Reply:
A very good point, should I replace these `assert!`s with `unwrap()` within the lines this PR touched, or do it globally 🤔

Reply:
If you want to, I'd really appreciate a global change, although I don't insist on it, because it can be quite tedious (for reference, the PR when I did it on Firecracker: https://github.com/firecracker-microvm/firecracker/pull/4321).

So just changing it for the function you touch here is fine!